Header header

# the ID of the user, is strictly greater than 0
int32 user_id

# the max nb of joints (does not need to be reached though, just a upper bound)
int8 SKEL_MAX_JOINTS = 25
kinect/NiteSkeletonJoint[] joints

